### PR TITLE
Glib: New version 2.83.2

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

File: G/Glib/build_tarballs.jl [hunk 1]
```diff
@@ -95,14 +95,14 @@ dependencies = [
     # Host gettext needed for "msgfmt"
     HostBuildDependency("Gettext_jll"),
     Dependency("Libiconv_jll"),
-    Dependency("Libffi_jll", v"3.2.2"; compat="~3.2.2"),
+    Dependency("Libffi_jll"; compat="~3.2.2"),
     # Gettext is only needed on macOS, as far as I could see
-    Dependency("Gettext_jll", v"0.21.0"; compat="=0.21.0"),
-    Dependency("PCRE2_jll"; compat="10.35"),
+    Dependency("Gettext_jll"; compat="=0.21.0"),
+    Dependency("PCRE2_jll"; compat="10.43"),
```
Comment (giordano) [inline G/Glib/build_tarballs.jl:101]: Pcre2 is a Julia dep

Comment (eschnett) [reply G/Glib/build_tarballs.jl:101]: Thanks.

**Comment by jwahlstrand** [comment]
Is there a reason to update to a development release? https://gitlab.gnome.org/GNOME/glib/-/blob/main/SECURITY.md#supported-versions: odd minor versions are considered development releases.

GLib is stable enough that I wouldn't necessarily expect problems, but Linux distributions tend to stick with even minor versions, which are the stable releases. Just to avoid potential issues I'll set a compat in Gtk4.jl to avoid this version if it's merged.

**Comment by giordano** [comment]
> Just to avoid potential issues I'll set a compat in Gtk4.jl to avoid this version if it's merged.

We need a compat syntax to exclude odd (and another one for even, just for symmetry) numbers :joy:

**Pull request closed without merging**